src/views/CountryNsOverviewContextAndStructure/NationalSocietyIndicators/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { useOutletContext } from 'react-router-dom';
+import { isDefined, isNotDefined } from '@togglecorp/fujs';
+
+import BlockLoading from '#components/BlockLoading';
+import Container from '#components/Container';
+import TextOutput from '#components/TextOutput';
+import useTranslation from '#hooks/useTranslation';
+import { type CountryOutletContext } from '#utils/outletContext';
+import { sumSafe } from '#utils/common';
+import { useRequest } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+function NationalSocietyIndicators() {
+    const strings = useTranslation(i18n);
+
+    const { countryId } = useOutletContext<CountryOutletContext>();
+
+    const {
+        pending: indicatorPending,
+        response: indicatorResponse,
+    } = useRequest({
+        url: '/api/v2/country/{id}/databank/',
+        skip: isNotDefined(countryId),
+        pathVariables: {
+            id: isDefined(countryId) ? Number(countryId) : undefined,
+        },
+    });
+
+    const youthValue = sumSafe([
+        indicatorResponse?.male_volunteer_age_6_12,
+        indicatorResponse?.male_volunteer_age_13_17,
+        indicatorResponse?.male_volunteer_age_18_29,
+        indicatorResponse?.female_volunteer_age_6_12,
+        indicatorResponse?.female_volunteer_age_13_17,
+        indicatorResponse?.female_volunteer_age_18_29,
+        indicatorResponse?.female_staff_age_18_29,
+        indicatorResponse?.male_staff_age_18_29,
+    ]);
```

Comment:
As per previous discussions, we should use the total value.
cc: @k9845
Indicators to use
```
KPI_PeopleVol_Tot_age_6_12
KPI_PeopleVol_Tot_age_13_17
KPI_PeopleVol_Tot_age_18_29
`